triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -1472,11 +1474,11 @@ object Runner {
     smap.expect("SMap", { case SMap(_, entries) if entries.contains(key) => entries(key) })
   }
 
-  private[trigger] def numberOfActiveContracts(
+  private[trigger] def getActiveContracts(
       svalue: SValue,
       level: Trigger.Level,
       version: Trigger.Version,
-  ): Option[Int] = {
+  ): Option[TreeMap[SValue, TreeMap[SValue, SValue]]] = {
```

Comment:
In trigger simulations, we need (on a per template basis) to count the number of active contracts - hence why we abstract out the `SMap` into the higher level `TreeMap`.